bundles/org.openhab.binding.tr064/README.md

```diff
@@ -0,0 +1,115 @@
+| `inboundCalls`             | `Number`                  |     x    | Number of inbound calls within the given number of days.       |
+| `macOnline`                | `Switch`                  |     x    | Online status of the device with the given MAC                 |
+| `missedCalls`              | `Number`                  |          | Number of missed calls within the given number of days.        |
+| `outboundCalls`            | `Number`                  |     x    | Number of outbound calls within the given number of days.      |
+| `reboot`                   | `Switch`                  |          | Reboot                                                         |
+| `rejectedCalls`            | `Number`                  |     x    | Number of rejected calls within the given number of days.      |
+| `securityPort`             | `Number`                  |     x    | The port for connecting via HTTPS to the TR-064 service.       |
+| `tamEnable`                | `Switch`                  |          | Enable/Disable the answering machine with the given index.     |
+| `tamNewMessages`           | `Number`                  |          | The number of new messages of the given answering machine.     |
+| `uptime`                   | `Number:Time`             |          | Uptime                                                         |
+| `wanAccessType`            | `String`                  |     x    | Access Type                                                    |
+| `wanConnectionStatus`      | `String`                  |          | Connection Status                                              |
+| `wanIpAddress`             | `String`                  |     x    | WAN IP Address                                                 |
+| `wanMaxDownstreamRate`     | `Number:DataTransferRate` |     x    | Max. Downstream Rate                                           |
+| `wanMaxUpstreamRate`       | `Number:DataTransferRate` |     x    | Max. Upstream Rate                                             |
+| `wanPhysicalLinkStatus`    | `String`                  |     x    | Link Status                                                    |
+| `wanTotalBytesReceived`    | `Number:DataAmount`       |     x    | Total Bytes Received                                           |
+| `wanTotalBytesSent`        | `Number:DataAmount`       |     x    | Total Bytes Send                                               |
+| `wifi24GHzEnable`          | `Switch`                  |          | Enable/Disable the 2.4 GHz WiFi device.                        |
+| `wifi5GHzEnable`           | `Switch`                  |          | Enable/Disable the 5.0 GHz WiFi device.                        |
+| `wifiGuestEnable`          | `Switch`                  |          | Enable/Disable the guest WiFi.                                 |
+
+## `PHONEBOOK` transformation
```

Comment:
```suggestion
## `PHONEBOOK` Transformation
```
... but shouldn't you better speak about a "profile" instead of a "transformation" here? I'd actually suggest to completely wipe the word "transformation" here (as well as from the code), since transformations are another concept in openHAB.

Reply:
agreed